Makefile

```diff
@@ -101,10 +101,15 @@ test-version: container
 # -----------
 
 .PHONY: test-e2e
-test-e2e: test-e2e-1.19
+test-e2e: test-e2e-1.20
+
 
 .PHONY: test-e2e-all
-test-e2e-all: test-e2e-1.19 test-e2e-1.18 test-e2e-1.17
+test-e2e-all: test-e2e-1.20 test-e2e-1.19 test-e2e-1.18 test-e2e-1.17
```

Comment:
Is 1.17 still in window of K8s supported releases?

Reply:
Indeed it isn't, I forgot to double check.